aeneas/src/vst/Parser.v3

```diff
@@ -196,30 +198,64 @@ component Parser {
 		}
 		return true;
 	}
-	def parsePacking(p: ParserState) -> VstPacking {
-		if (p.curByte == '0') {
-			p.eat1();
-			if (p.curByte == 'b' || p.curByte == 'B') p.eat1();
-			else p.error("expected bit pattern");
+	def parsePackingParam(p: ParserState) -> (Token, u8) {
+		var param = parseIdentVoid(p).name;
+		p.req1(':');
 
-			var s = p.curPos, q = p.curPos;
+		var sizeLiteral = parseIntLiteral(p);
+		var size = u8.!(V3.unboxI32(sizeLiteral.val));
+		return (param, size);
+	}
+	def parsePackingSize(p: ParserState) -> u8 {
+		var sizeLiteral = parseIntLiteral(p);
+		if (sizeLiteral == null) {
+			p.error("size expected");
+			return 0;
+		}
+		else return u8.!(V3.unboxI32(sizeLiteral.val));
```

Comment:
I think we want to keep this as a literal, so that it can be checked in the Verifier that it's a reasonable size.